src/PrestaShopBundle/Resources/views/Admin/Improve/International/Currency/Blocks/form.html.twig

```diff
@@ -0,0 +1,78 @@
+{#**
+ * 2007-2018 PrestaShop
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to http://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2018 PrestaShop SA
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ *#}
+
+{% import '@PrestaShop/Admin/macros.html.twig' as ps %}
+
+{% block form %}
+  {{ form_start(currencyForm) }}
+
+  <div class="card">
+    <div class="card-header">
+      {{ 'Currencies'|trans({}, 'Admin.Global') }}
+    </div>
+    <div class="card-block row">
+      <div class="card-text">
+
+        {{ ps.form_group_row(currencyForm.iso_code, {}, {
+          'label': 'Currency'|trans({}, 'Admin.Global'),
+          'help': 'ISO code (e.g. USD for Dollars, EUR for Euros, etc.).'|trans({}, 'Admin.International.Help')
+        }) }}
+
+        {% set exchangeRateLabel %}
+          {{ 'Exchange rates are calculated from one unit of your shop\'s default currency. For example, if the default currency is euros and your chosen currency is dollars, type "1.20" (1&euro; = $1.20).'|trans({}, 'Admin.International.Help') }}
```

Comment:
@tomas862 Pablo notified me about this wording as `1&euro;` is being printed "as is" :
![capture d ecran 2019-02-12 a 17 51 21](https://user-images.githubusercontent.com/3830050/52653390-2c31ca00-2ef0-11e9-9193-77deb67152b1.png)

I cant find it in legacy page. Did you create it ?

Reply:
Yes it existed previously on `AdminCurrenciesController.php` line 119 - this file is currently removed due to whole page is migrated. I agree this looks strange - maybe and update here required for the transaltion?